Utilities/BEType.h

```diff
@@ -490,6 +490,11 @@ class se_t<T, true, Align>
 		return storage::from(m_data);
 	}
 
+	type raw() const
```

Comment:
It should just return m_data. Preferably reference to, like in atomic_t. Casting it to the type is not meaningful.